src/platforms/Print.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <stdint.h>
+
+#ifndef NEO_GPS_PRINT
+  #define NEO_GPS_PRINT NeoGPS::Print
+  #ifdef __GNUC__
+    #define NEO_GPS_PRINT_DEFAULT_IMPL_WARN \
+    __attribute__((deprecated("You are using a function which uses NEO_GPS_PRINT without defining the type.  Compilation will fail.  To fix this #define NEO_GPS_PRINT your_print_type_t  before including platform.h")))
+  #endif
+
+/**
+ * This file implements a non working dummy implemenation for Print
+ **/
+namespace NeoGPS {
+  
+  class Print {
+  };
+  
+  Print & operator << ( Print & print, char ) { return print; }
+  Print & operator << ( Print & print, uint8_t ) { return print; }
```

Comment:
this function is not necessary and should be removed.

`uint8_t` is usually `typedef`'d to `unsigned char` and for instance `std::cout` will not print a number but the corresponding ascii char.

I've added a + in front of all `uint8_t` prints which automatically casts them them to `int`.

Reply:
`std::cout` prints the ASCII character for `unsigned char`?  How are you supposed to print the number?  `unsigned short`?  Cast it to `int`?  Hmmm...

Reply:
Yes.  You need to cast.
The easiest way to cast is by adding + in front of an uint8_t
`std::cout << n;`  becomes `std::cout << +n;`  This will cast uint8_t to ints.  All other types stay the same.  I've added a `+` in front of all `<< aNumber`.  Regardless of the actual number type.  All numbers are now output as `outs << +aNumber;`

Reply:
Doh!  Now your previous comment makes sense:

>I've added a + in front of all uint8_t prints which automatically casts them them to int.

I was totally derailed by the source code diffs above that show "+" at the beginning of the line...